### ISSUE TITLE
Action fails: parse error: Invalid numeric literal at line 1, column 6

### ISSUE THREAD

**Issue by lukehinds** [CodelyTV/pr-size-labeler#59]
I get several failures and the label is not assigned:

```
Usage: basename FILE [SUFFIX]

Strip directory path and .SUFFIX from FILE
parse error: Invalid numeric literal at line 1, column 4
/src/github.sh: line 29: ((: changes += : syntax error: operand expected (error token is "+= ")
parse error: Invalid numeric literal at line 1, column 6
BusyBox v1.34.1 (2022-07-19 20:11:24 UTC) multi-call binary.

Usage: basename FILE [SUFFIX]

Strip directory path and .SUFFIX from FILE
parse error: Invalid numeric literal at line 1, column 6
/src/github.sh: line 29: ((: changes += : syntax error: operand expected (error token is "+= ")
Total modifications (additions + deletions): 0
Ignoring files (if present): "pkg/generated/*""docs/*""database/mock/store.go""go.mod""go.sum"
Labeling pull request with size/xs
jq: error (at <stdin>:1): Cannot iterate over null (null)
Final labels: "size/xs"
```

**Comment by crishellco** [comment]
I'm seeing the same thing:

![image](https://github.com/CodelyTV/pr-size-labeler/assets/1878509/3e47a09b-7da2-4b6a-89df-cf0fd48acbcf)

**Comment by lukehinds** [comment]
I stopped using this action, its too unstable and prone to breaking.

**Comment by Tuizi** [comment]
I guess this action is not maintained anymore 🤷

**Comment by EnyMan** [comment]
Check your github api url it should be `https://api.github.com` and not `api.github.com`.

**Comment by Tuizi** [comment]
Are you sure @EnyMan? using `https://` give the error: `curl: (3) URL rejected: Malformed input to a URL function`

**Comment by EnyMan** [comment]
@Tuizi Yes we have it set to `github_api_url: 'https://api.github.com'` and it works

**Comment by johnlk** [comment]
> I stopped using this action, its too unstable and prone to breaking.

Take a look at the latest version `v.1.10.0`. I thought the same thing about the project not being maintained, but I was able to connect with the maintainers and we pushed a lot of fixes recently and a couple features.

**Comment by johnlk** [comment]
This issue was resolved with #62 and #57
